config/core/resources/service.yaml

```diff
@@ -23,24 +23,41 @@ metadata:
     duck.knative.dev/podspecable: "true"
 spec:
   group: serving.knative.dev
-  preserveUnknownFields: false
-  validation:
-    openAPIV3Schema:
-      type: object
-      # this is a work around so we don't need to flush out the
-      # schema for each version at this time
-      #
-      # see issue: https://github.com/knative/serving/issues/912
-      x-kubernetes-preserve-unknown-fields: true
   versions:
-  - name: v1alpha1
+  - &version
+    name: v1alpha1
     served: true
     storage: false
-  - name: v1beta1
-    served: true
-    storage: false
-  - name: v1
-    served: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        # this is a work around so we don't need to flush out the
+        # schema for each version at this time
+        #
+        # see issue: https://github.com/knative/serving/issues/912
+        x-kubernetes-preserve-unknown-fields: true
+    additionalPrinterColumns:
+    - name: URL
+      type: string
+      jsonPath: .status.url
+    - name: LatestCreated
+      type: string
+      jsonPath: .status.latestCreatedRevisionName
+    - name: LatestReady
+      type: string
+      jsonPath: .status.latestReadyRevisionName
+    - name: Ready
+      type: string
+      jsonPath: ".status.conditions[?(@.type=='Ready')].status"
+    - name: Reason
+      type: string
+      jsonPath: ".status.conditions[?(@.type=='Ready')].reason"
+  - << : *version
+    name: v1beta1
+  - << : *version
+    name: v1
```

Comment:
@dprotaso hey Dave, did I overlook this or, did you add it to the commit? :thinking: 

I have not that on eventing :sweat_smile:

Reply:
This is just yaml hackery that merges v1alpha1 content into this yaml node and I override the `name` and `storage` properties